CHANGELOG.md

```diff
@@ -4,14 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# v2.13.7
+# [Unreleased]
 
 ### Added
+- Addresses #175.
```

Comment:
this line isn't necessary but if you want to put it, hyperlink to original issue

Reply:
Removed.